programs/scope/src/handlers/handler_update_token_metadata.rs

```diff
@@ -0,0 +1,59 @@
+use crate::{OracleMappings, ScopeError, VALUE_BYTE_ARRAY_LEN};
+use anchor_lang::prelude::*;
+use num_enum::TryFromPrimitive;
+
+#[derive(Accounts)]
+#[instruction(token:usize, price_type: u8, feed_name: String)]
+pub struct UpdateTokensMetadata<'info> {
+    pub admin: Signer<'info>,
+    #[account(seeds = [b"conf", feed_name.as_bytes()], bump, has_one = admin, has_one = oracle_mappings, has_one = tokens_metadata)]
+    pub configuration: AccountLoader<'info, crate::Configuration>,
+    #[account(mut)]
+    pub oracle_mappings: AccountLoader<'info, OracleMappings>,
+    /// CHECK: We trust the admin to provide a trustable account here. Some basic sanity checks are done based on type
+    pub price_info: AccountInfo<'info>,
+
+    #[account(mut)]
+    pub tokens_metadata: AccountLoader<'info, crate::TokensMetadata>,
+}
+
+pub fn process(
+    ctx: Context<UpdateTokensMetadata>,
+    index: u64,
+    mode: u64,
+    value: &[u8; VALUE_BYTE_ARRAY_LEN],
```

Comment:
Dynamic could work here.
```suggestion
    value: Vec<u8>,
```